ext/openssl/extconf.rb

```diff
@@ -174,6 +174,11 @@ def find_openssl_library
 have_func("EVP_PBE_scrypt")
 have_func("SSL_CTX_set_post_handshake_auth")
 
+# added in 1.1.1
+have_func("EC_POINT_get_affine_coordinates")
+have_func("EC_POINT_set_affine_coordinates")
+have_func("EC_POINT_set_compressed_coordinates")
```

Comment:
Can we have an alternate implementation in `ext/openssl/openssl_missing.c` using the suffixed variants? `EC_POINT_get_affine_coordinates()` only exists in OpenSSL 1.1.1, and the latest LibreSSL still doesn't have it.

Checking only `EC_POINT_get_affine_coordinates` should be good enough since it's unlikely a future LibreSSL version implements just one or two of the three functions.

Reply:
"alternate implementation" as in using the GFp and GF2m functions, or my nonsense encoding kludge?

Reply:
I meant using the GFp and GF2m functions. They should both exist in OpenSSL <= 1.1.0 and LibreSSL.